### PR TITLE
Feat: Spacer Block

### PR DESCRIPTION
**Pull request by blankart**
#### Changes
- No new API introduced

### CODE REVIEW

File: src/block/spacer/style.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+const getStyleParams = () => {
+	return [
+		{
+			renderIn: 'save',
+			selector: '',
+			styleRule: 'height',
+			attrName: 'height',
+			format: '%spx',
+		},
```
Comment (blankart) [inline src/block/spacer/style.js:23]:
```suggestion
			attrName: 'height',
			format: '%spx',
			responsive: 'all',			
		},
```

**Review by bfintal — CHANGES_REQUESTED** [review]
When you insert the block, the block has 0 height. It should default to `50px`

**Review by bfintal — CHANGES_REQUESTED** [review]
1. The resizer should only be visible when the block is hovered on
2. When the resizer is dragged and you save the post, refreshing the editor will show an error

**Review by bfintal — CHANGES_REQUESTED** [review]
A few more things that we should add:
- [x] Our image block snaps to specific widths (multiples of 50px) when resizing, we should add this here as well
- [x] When resizing, there is no indicator of the current size of the spacer, it might be a good idea to add a tooltip similar to the image block too that's visible when hovering over the spacer block

**Review by bfintal — CHANGES_REQUESTED** [review]
Some comments:
- [ ] The orange color when dragging looks off, make it transparent
- [ ] When you change the height, if you preview in tablet and mobile, the height goes back to default in the editor (works okay in frontend)
- [ ] The popup height shows the reset button when it's already at the default value. Either the reset button shouldn't be there when the value is the default, or the value should show blank when it's the default to prevent the reset button from showing up

**Pull request merged**